doc/UsersGuide/source/doc-snippets/hsd_run_multiple.rst

```diff
@@ -0,0 +1,6 @@
+To run multiple cases at once, copy ``test_cases.yaml`` from the test cases directory and use the ``-l`` argument:
```

Comment:
test_cases.yaml was removed from the codebase. users can simply modify `ufs_test.yaml` to contain a subset of tests (i.e. 2020_CAPE and baroclinic_wave) by deleting the others, or copying the sections for 2020_CAPE/baroclinic wave into a new *.yaml file to call via ufs_test.sh.